From 7eda61191e688c662f0bf756334c3d06f900adc3 Mon Sep 17 00:00:00 2001
From: Miquel Raynal <miquel.raynal@bootlin.com>
Date: Thu, 18 Oct 2018 15:19:06 +0200
Subject: [PATCH 0719/1051] ARM64: dts: a3900-db: add NAND pinctrl information

Ensure the muxing around the NAND controller is correct (do not rely
on the bootloader's configuration) by adding a reference on the
-already defined- NAND pins groups.

Change-Id: I62b21bb0056b28b80934a024585b3d2ed2326a31
Signed-off-by: Miquel Raynal <miquel.raynal@bootlin.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60388
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
index d08b07106594..4870efa18504 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
@@ -95,6 +95,9 @@
 };
 
 &cp0_nand_controller {
+	pinctrl-names = "default";
+	pinctrl-0 = <&nand_pins &nand_rb>;
+
 	nand@0 {
 		reg = <0>;
 		label = "main-storage";
-- 
2.17.1

